server/src/main/java/org/candlepin/async/JobManager.java

```diff
@@ -1126,4 +1137,50 @@ public void modeChanged(Mode mode) {
             }
         }
     }
+
+    /**
+     * Lookup job status by job id.
+     *
+     * @param jobId
+     *  Id of the requested job
+     *
+     * @return
+     *  {@link AsyncJobStatus} of the requested job
+     *
+     * @throws JobAccessException.NotFound
+     *  When requested job was not found
+     */
+    public AsyncJobStatus get(final String jobId) throws JobAccessException {
```

Comment:
The naming convention to follow is `<verb><noun><conjunction><modifiers>`. Keeping in step with the other methods, these should be getJob and cancelJob.

Reply:
Done.